run-devel.sh

```diff
@@ -58,6 +58,8 @@ if [ ! "$(docker ps -q -f name=$CONTAINER)" ]; then
             --volume="/etc/sudoers.d:/etc/sudoers.d:ro" \
             --volume="/tmp/.X11-unix:/tmp/.X11-unix:rw" \
             --privileged \
+            --gpus all \
+            --env="NVIDIA_DRIVER_CAPABILITIES=utility,graphics,compute" \
```

Comment:
How about:
'all,"capabilities=utility,graphics,compute"'?

Reply:
I tried something like this before but I had some problems. I can try again with your suggestions.

Reply:
Implemented in 820aa2c